xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/LiveTableGenerator.xml

```diff
@@ -128,6 +128,10 @@ ${escapetool.h}set (${escapetool.d}columnsProperties = {
 #set($userPicker = true)
 ${escapetool.h}set (${escapetool.d}discard = ${escapetool.d}xwiki.ssfx.use('uicomponents/widgets/userpicker/userPicker.css'))
 #end
+#if($column == 'doc.author')
```

Comment:
It does not make much sense to have two different #if with the exact same content. You should merge them.

Reply:
Ikr I already committed it.